web/profiles/joinup/joinup.behat.inc

```diff
@@ -0,0 +1,163 @@
+<?php
+
+/**
+ * @file
+ * Contains \UserSubContext.
+ */
+
+use Behat\Gherkin\Node\TableNode;
+use Drupal\DrupalExtension\Context\DrupalSubContextBase;
+use Drupal\DrupalExtension\Context\DrupalSubContextInterface;
+use \Drupal\user\Entity\User;
+use \Drupal\file\Entity\File;
+
+/**
+ * Behat step definitions for testing users.
+ */
+class UserSubContext extends DrupalSubContextBase implements DrupalSubContextInterface {
+
+  /**
+   * Test users.
+   *
+   * @var \Drupal\user\Entity\User[]
+   */
+  protected $users = [];
+
+  /**
+   * Test images.
+   *
+   * @var \Drupal\file\Entity\File[]
+   */
+  protected $images = [];
+
+  /**
+   * Creates a user with data provided in a table.
+   *
+   * The following fields are required:
```

Comment:
Maybe change this to table format as it is in modules/custom/collection/collection.behat.inc marking the required fields with an asterisk or something similar.
This is only to keep the same way of writing comments and keep it generic rather than personalized comments.